connectors/sources/tests/test_generic_database.py

```diff
@@ -155,8 +90,32 @@ def fetchmany(self, size):
         return []
 
     def fetchall(self):
-        """This method returns object of Return class"""
-        return [(10,)]
+        """This method returns results of query"""
+        self.query = str(self.query)
+        query_object_mssql = MSSQLQueries()
```

Comment:
should probably not live inside in `test_generic_database.py`. This is specific for the corresponding data source.

Reply:
You mean `CursorSync` class? If yes, It is common for `test_oracle` and `test_mssql`. So, we put it in test_generic_database.py file.

Reply:
To reuse the queries I created an object of `MSSQLQueries` class. Otherwise, we need to use hardcoded queries value.

Reply:
I think you should separate the faking of the query results for every datasource and push the corresponding classes down to the concrete tests (`test_mssql.py` and `test_oracle.py`). It's probably also better to just mock the methods inside the concrete datasource tests without creating a dedicated class.

Reply:
I see too many `if...else...` in the dummy class. Agree to Tim that we should move them to specific test files and remove the `if...else...`, or even better, we can use mock instead of defining dummy class.

Reply:
As we have multiple functions it's better to use a dummy class.